typed_environment_configuration/typed_environment_configuration.py

```diff
@@ -61,7 +59,7 @@ def _create_type_checker(self):
         return typepy.checker.StringTypeChecker(self._data, self._strict_level)
 
     def _create_type_converter(self):
-        return EnvListConverter(self._data)
+        return EnvListConverter(self._data, self._params)
```

Comment:
Uses new signature of `typepy.type.List.__init__`: https://github.com/thombashi/typepy/blob/6340af849c1202cf758dc588f7b669ded99033a9/typepy/type/_list.py#L34